test/fixtures/fixture-sharepointer.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function(){
```

Comment:
Good to know, I like the stub functions - just results in a lot more duplicated stubbing code if it's not in a fixture? Supercore tests are monstrous sized because of this